### PR TITLE
add explaination for log densitive feat

### PR DESCRIPTION
**Pull request by xhebox**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -4141,10 +4141,12 @@ EXPLAIN FORMAT='brief' SELECT COUNT(1) FROM t WHERE a = 1 AND b IS NOT NULL;
...
 - 是否持久化到集群：是
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
-- 类型：布尔型
+- 类型：枚举型
 - 默认值：`OFF`
+- 可选值：`OFF`、`ON`、`MARKER`。
 - 这个变量用于控制在记录 TiDB 日志和慢日志时，是否将 SQL 中的用户信息遮蔽。
-- 将该变量设置为 `1` 即开启后，假设执行的 SQL 为 `insert into t values (1,2)`，在日志中记录的 SQL 会是 `insert into t values (?,?)`，即用户输入的信息被遮蔽。
+- 将该变量设置为 `ON` 后，假设执行的 SQL 为 `insert into t values (1,2)`，在日志中记录的 SQL 会是 `insert into t values (?,?)`，即用户输入的信息被遮蔽。
+- 将该变量设置为 `MARKER` 后，假设执行的 SQL 为 `insert into t values (1,2)`，在日志中记录的 SQL 会是 `insert into t values (‹1›,‹2›)`。
```
Comment (hfxsd) [inline system-variables.md:4149]:
```suggestion
- 将该变量设置为 `ON` 后，假设执行的 SQL 为 `INSERT INTO t VALUES (1,2)`，在日志中记录的 SQL 会是 `INSERT INTO t VALUES (?,?)`，即用户输入的信息被遮蔽。
- 将该变量设置为 `MARKER` 后，假设执行的 SQL 为 `INSERT INTO t VALUES (1,2)`，在日志中记录的 SQL 会是 `INSERT INTO t VALUES (‹1›,‹2›)`。
```

File: system-variables.md [hunk 2]
```diff
@@ -4141,10 +4141,12 @@ EXPLAIN FORMAT='brief' SELECT COUNT(1) FROM t WHERE a = 1 AND b IS NOT NULL;
...
 - 是否持久化到集群：是
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
-- 类型：布尔型
+- 类型：枚举型
 - 默认值：`OFF`
+- 可选值：`OFF`、`ON`、`MARKER`。
 - 这个变量用于控制在记录 TiDB 日志和慢日志时，是否将 SQL 中的用户信息遮蔽。
-- 将该变量设置为 `1` 即开启后，假设执行的 SQL 为 `insert into t values (1,2)`，在日志中记录的 SQL 会是 `insert into t values (?,?)`，即用户输入的信息被遮蔽。
+- 将该变量设置为 `ON` 后，假设执行的 SQL 为 `INSERT INTO t VALUES (1,2)`，在日志中记录的 SQL 会是 `INSERT INTO t VALUES (?,?)`，即用户输入的信息被遮蔽。
+- 将该变量设置为 `MARKER` 后，假设执行的 SQL 为 `INSERT INTO t VALUES (1,2)`，在日志中记录的 SQL 会是 `INSERT INTO t VALUES (‹1›,‹2›)`。
```
Comment (hfxsd) [inline system-variables.md:4149]:
```suggestion
- 将该变量设置为 `MARKER` 后，假设执行的 SQL 为 `INSERT INTO t VALUES (1,2)`，在日志中记录的 SQL 会是 `INSERT INTO t VALUES (‹1›,‹2›)`，即用户输入的信息被标记符号取代。
```

Comment (xhebox) [reply system-variables.md:4149]: 是包裹，不是取代。

Comment (xhebox) [reply system-variables.md:4149]: 想起来还有个新改动没写上去： `‹`会转义成 `‹‹`, `›`同理.

Comment (hfxsd) [reply system-variables.md:4149]: Ok updated

File: system-variables.md [hunk 3]
```diff
@@ -4141,10 +4141,12 @@ EXPLAIN FORMAT='brief' SELECT COUNT(1) FROM t WHERE a = 1 AND b IS NOT NULL;
 - 作用域：SESSION | GLOBAL
 - 是否持久化到集群：是
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
-- 类型：布尔型
+- 类型：枚举型
 - 默认值：`OFF`
+- 可选值：`OFF`、`ON`、`MARKER`。
 - 这个变量用于控制在记录 TiDB 日志和慢日志时，是否将 SQL 中的用户信息遮蔽。
-- 将该变量设置为 `1` 即开启后，假设执行的 SQL 为 `insert into t values (1,2)`，在日志中记录的 SQL 会是 `insert into t values (?,?)`，即用户输入的信息被遮蔽。
+- 将该变量设置为 `ON` 后，假设执行的 SQL 为 `INSERT INTO t VALUES (1,2)`，在日志中记录的 SQL 会是 `INSERT INTO t VALUES (?,?)`，即用户输入的信息被遮蔽。
```
Comment (ljun0712) [inline system-variables.md:4148]: 建议补齐OFF的说明

Comment (hfxsd) [reply system-variables.md:4148]: Updated

File: system-variables.md [hunk 4]
```diff
@@ -4141,10 +4141,13 @@ EXPLAIN FORMAT='brief' SELECT COUNT(1) FROM t WHERE a = 1 AND b IS NOT NULL;
...
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
-- 类型：布尔型
+- 类型：枚举型
 - 默认值：`OFF`
+- 可选值：`OFF`、`ON`、`MARKER`。
 - 这个变量用于控制在记录 TiDB 日志和慢日志时，是否将 SQL 中的用户信息遮蔽。
-- 将该变量设置为 `1` 即开启后，假设执行的 SQL 为 `insert into t values (1,2)`，在日志中记录的 SQL 会是 `insert into t values (?,?)`，即用户输入的信息被遮蔽。
+- 默认值为 `OFF`，即对用户输入的信息不做任何处理。
+- 将该变量设置为 `ON` 后，假设执行的 SQL 为 `INSERT INTO t VALUES (1,2)`，在日志中记录的 SQL 会是 `INSERT INTO t VALUES (?,?)`，即用户输入的信息被遮蔽。
+- 将该变量设置为 `MARKER` 后，假设执行的 SQL 为 `INSERT INTO t VALUES (1,2)`，在日志中记录的 SQL 会是 `INSERT INTO t VALUES (‹1›,‹2›)`，即用户输入的信息被标记符号 `‹ ›` 包裹。输入信息中的 `‹` 会转义成 `‹‹`、`›` 会转义成 `››`。基于标记后的日志，你可以在日志展示时决定定是否对标记信息进行脱敏处理。
```
Comment (hfxsd) [inline system-variables.md:4150]:
```suggestion
- 将该变量设置为 `MARKER` 后，假设执行的 SQL 为 `INSERT INTO t VALUES (1,2)`，在日志中记录的 SQL 会是 `INSERT INTO t VALUES (‹1›,‹2›)`，即用户输入的信息被标记符号 `‹ ›` 包裹。输入信息中的 `‹` 会转义成 `‹‹`、`›` 会转义成 `››`。基于标记后的日志，你可以在日志展示时决定是否对标记信息进行脱敏处理。
```

File: system-variables.md [hunk 5]
```diff
@@ -4141,10 +4141,13 @@ EXPLAIN FORMAT='brief' SELECT COUNT(1) FROM t WHERE a = 1 AND b IS NOT NULL;
...
 - 是否持久化到集群：是
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
-- 类型：布尔型
+- 类型：枚举型
 - 默认值：`OFF`
+- 可选值：`OFF`、`ON`、`MARKER`。
 - 这个变量用于控制在记录 TiDB 日志和慢日志时，是否将 SQL 中的用户信息遮蔽。
-- 将该变量设置为 `1` 即开启后，假设执行的 SQL 为 `insert into t values (1,2)`，在日志中记录的 SQL 会是 `insert into t values (?,?)`，即用户输入的信息被遮蔽。
+- 默认值为 `OFF`，即对用户输入的信息不做任何处理。
+- 将该变量设置为 `ON` 后，假设执行的 SQL 为 `INSERT INTO t VALUES (1,2)`，在日志中记录的 SQL 会是 `INSERT INTO t VALUES (?,?)`，即用户输入的信息被遮蔽。
```
Comment (hfxsd) [inline system-variables.md:4149]:
```suggestion
- 将该变量设置为 `ON` 后，用户输入的信息被遮蔽。假设执行的 SQL 为 `INSERT INTO t VALUES (1,2)`，在日志中记录的 SQL 会是 `INSERT INTO t VALUES (?,?)`。
```

File: system-variables.md [hunk 6]
```diff
@@ -4141,10 +4141,13 @@ EXPLAIN FORMAT='brief' SELECT COUNT(1) FROM t WHERE a = 1 AND b IS NOT NULL;
...
+- 将该变量设置为 `MARKER` 后，假设执行的 SQL 为 `INSERT INTO t VALUES (1,2)`，在日志中记录的 SQL 会是 `INSERT INTO t VALUES (‹1›,‹2›)`，即用户输入的信息被标记符号 `‹ ›` 包裹。输入信息中的 `‹` 会转义成 `‹‹`、`›` 会转义成 `››`。基于标记后的日志，你可以在日志展示时决定是否对标记信息进行脱敏处理。
```
Comment (hfxsd) [inline system-variables.md:4150]:
```suggestion
- 将该变量设置为 `MARKER` 后，用户输入的信息被标记符号 `‹ ›` 包裹。假设执行的 SQL 为 `INSERT INTO t VALUES (1,2)`，在日志中记录的 SQL 会是 `INSERT INTO t VALUES (‹1›,‹2›)`。输入信息中的 `‹` 会转义成 `‹‹`、`›` 会转义成 `››`。基于标记后的日志，你可以在日志展示时决定是否对标记信息进行脱敏处理。
```

File: system-variables.md [hunk 7]
```diff
@@ -4141,10 +4141,13 @@ EXPLAIN FORMAT='brief' SELECT COUNT(1) FROM t WHERE a = 1 AND b IS NOT NULL;
 - 作用域：SESSION | GLOBAL
 - 是否持久化到集群：是
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
-- 类型：布尔型
+- 类型：枚举型
 - 默认值：`OFF`
+- 可选值：`OFF`、`ON`、`MARKER`。
```
Comment (Oreoxmt) [inline system-variables.md:4146]:
```suggestion
- 可选值：`OFF`、`ON`、`MARKER`
```

File: system-variables.md [hunk 8]
```diff
@@ -4141,10 +4141,13 @@ EXPLAIN FORMAT='brief' SELECT COUNT(1) FROM t WHERE a = 1 AND b IS NOT NULL;
...
 - 这个变量用于控制在记录 TiDB 日志和慢日志时，是否将 SQL 中的用户信息遮蔽。
-- 将该变量设置为 `1` 即开启后，假设执行的 SQL 为 `insert into t values (1,2)`，在日志中记录的 SQL 会是 `insert into t values (?,?)`，即用户输入的信息被遮蔽。
+- 默认值为 `OFF`，即对用户输入的信息不做任何处理。
+- 将该变量设置为 `ON` 后，用户输入的信息被遮蔽。假设执行的 SQL 为 `INSERT INTO t VALUES (1,2)`，在日志中记录的 SQL 会是 `INSERT INTO t VALUES (?,?)`。
+- 将该变量设置为 `MARKER` 后，用户输入的信息被标记符号 `‹ ›` 包裹。假设执行的 SQL 为 `INSERT INTO t VALUES (1,2)`，在日志中记录的 SQL 会是 `INSERT INTO t VALUES (‹1›,‹2›)`。输入信息中的 `‹` 会转义成 `‹‹`、`›` 会转义成 `››`。基于标记后的日志，你可以在日志展示时决定是否对标记信息进行脱敏处理。
```
Comment (hfxsd) [inline system-variables.md:4151]:
```suggestion

> **警告：**
>
> `MARKER` 选项目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。

```

File: system-variables.md [hunk 9]
```diff
@@ -4141,10 +4141,13 @@ EXPLAIN FORMAT='brief' SELECT COUNT(1) FROM t WHERE a = 1 AND b IS NOT NULL;
...
```
Comment (Oreoxmt) [inline system-variables.md:4150]:
```suggestion
- 将该变量设置为 `ON` 后，用户输入的信息被遮蔽。假设执行的 SQL 为 `INSERT INTO t VALUES (1,2)`，则日志中记录的 SQL 语句为 `INSERT INTO t VALUES (?,?)`。
- 将该变量设置为 `MARKER` 后，用户输入的信息被标记符号 `‹ ›` 包裹。假设执行的 SQL 为 `INSERT INTO t VALUES (1,2)`，则日志中记录的 SQL 语句为 `INSERT INTO t VALUES (‹1›,‹2›)`。输入信息中的 `‹` 会转义成 `‹‹`，`›` 会转义成 `››`。基于标记后的日志，你可以在展示日志时决定是否对被标记信息进行脱敏处理。
```

**Review by hfxsd — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**